src/khoj/processor/content/text_to_entries.py

```diff
@@ -148,10 +148,10 @@ def update_embeddings(
                 hashes_to_process |= hashes_for_file - existing_entry_hashes
 
         embeddings = []
+        model = get_default_search_model()
```

Comment:
Should index also use get_user_default_search_model? It'll avoid mismatched embedding model usage for generating embedding for user data and user query

```suggestion
        model = get_user_default_search_model()
```